### ISSUE TITLE
Question about Enhance-iCurb loss

### ISSUE THREAD

**Issue by Huijie2020** [TonyXuQAQ/Topo-boundary#3]
Hi,

I am training Enhance-iCurb baseline. After decreasing batch size and learning rate, the model doesn't converge.

In the first model, I changed `batch_size: 64` and `lr_rate: 0.00005`(Default setting : `batch_size: 128` and `lr_rate: 0.0001`. Both coor_loss and stop_loss don't converge.

In the 2nd model, I changed `batch_size: 64` and `lr_rate: 0.00001`(Default setting : `batch_size: 128` and `lr_rate: 0.0001`. Stop_loss doesn't converge, but the lr is very small.

![image](https://user-images.githubusercontent.com/62360605/120100649-a654df80-c0f6-11eb-83bc-16fb41eec62d.png)

Should I continue to decrease the learning rate to 0.000001?

Thank you!

**Comment by TonyXuQAQ** [comment]
In our experiments, the variance of the stop_loss is also huge, for both iCurb and Enhanced-iCurb. You could ignore this phenomenon at this stage. As long as the coord_loss converges, you should get an acceptable result. Set ```lr_rate``` around 0.0001. 
![Screenshot from 2021-06-02 09-52-35](https://user-images.githubusercontent.com/33851435/120413743-74a87680-c38b-11eb-8201-6f6eee56bb7a.png)

**Comment by Huijie2020** [comment]
Thank you for the reply.

I set `batch_size: 64` and `lr_rate: 0.00001`, and got the results which are very close to the results from the provided checkpoints. I may try `lr_rate: 0.0001` in the future.
![a](https://user-images.githubusercontent.com/62360605/120415853-1474ee80-c311-11eb-8af9-8b2a4a6f8c6e.png)
![b](https://user-images.githubusercontent.com/62360605/120415877-1f2f8380-c311-11eb-89d5-c8aa717dc89c.png)

**Comment by Huijie2020** [comment]
I am closing the issue.
